CodeKata/TwiceAsOldSolution.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CodeKata
+{
+    public class TwiceAsOldSolution
+    {
+        public static int TwiceAsOld(int dadYears, int sonYears)
+        {
+            return dadYears - 2 * sonYears > 0 ? dadYears - 2 * sonYears : (dadYears - 2 * sonYears) * (-1);
```

Comment:
You just duplicated same code 3 times in one liner, nice:
```
dadYears - 2 * sonYears
```
Maybe try to refactor piece of code above and give it a meaningfull name?
You can also use System.Math.Abs static method instead (https://learn.microsoft.com/pl-pl/dotnet/api/system.math.abs?view=net-6.0):
```
using System;

return Math.Abs(dadYears - 2 * sonYears);
```